src/frontend/src/lib/constants/test-ids.constants.ts

```diff
@@ -52,6 +54,7 @@ export const NAVIGATION_ITEM_ACTIVITY = 'navigation-item-activity';
 export const NAVIGATION_ITEM_EXPLORER = 'navigation-item-explore';
 export const NAVIGATION_ITEM_SETTINGS = 'navigation-item-settings';
 export const NAVIGATION_ITEM_HOMEPAGE = 'navigation-item-homepage';
+export const NAVIGATION_ITEM_MANAGE_LIST = 'navigation-item-manage-list';
```

Comment:
this is not exactly a navigation item right? What do you think of...

```suggestion
export const MANAGE_TOKENS_MODAL_BUTTON = 'manage-tokens-modal-button';
```

and put it together with `MANAGE_TOKEN_LIST_SAVE` in a separate session that pertains to MANAGE modal